### ISSUE TITLE
关于bge-reranker-v2*最长长度的问题

### ISSUE THREAD

**Issue by Zhouziyi828** [FlagOpen/FlagEmbedding#932]
您好，按照我的理解，v2应该支持最长长度8192的输入，但是我在FlagEmbeeding的调用方式中看到：
class LayerWiseFlagLLMReranker:
@torch.no_grad()
    def compute_score(self, sentence_pairs: Union[List[Tuple[str, str]], Tuple[str, str]], batch_size: int = 16,
                      max_length: int = 512, cutoff_layers: List[int] = None, prompt: str = None,
                      normalize: bool = False, use_dataloader: bool = False,
                      num_workers: int = None) -> Union[float, List[float], List[List[float]]]:
默认参数设置的是512，但是您在readme里没有强调我们需要再compute_score时指定该参数。如果我们不指定，是否就会按照512进行截断？

**Comment by staoxiao** [comment]
是的，不指定的话，会按照512截断。
主要是由于8192长度需要较大的显存才能推理，且一般较少场景会用到，所以默认使用512。

**Comment by Zhouziyi828** [comment]
好的明白。谢谢您的回复

**Comment by staoxiao** [comment]
训练后的模型还是支持8192的，但可能性能会有损失。如果下游任务有很长的数据，那还是最好训练数据中也有一些比较长的数据，同时设置更大的训练长度，让模型训练更充分。

**Comment by Zhouziyi828** [comment]
明白了 谢谢您的解答
